### DISCUSSION TITLE
Theme for bat

### DISCUSSION THREAD

**Discussion by NeckBeardPrince** [diegoulloao/neofusion.nvim#3] [Ideas]
https://github.com/sharkdp/bat

**Comment by diegoulloao** [comment]
Definitely the theme for `bat` is on the list.

**Comment by diegoulloao** [comment]
@NeckBeardPrince https://github.com/diegoulloao/neofusion.bat/

**Reply by NeckBeardPrince** [reply]
Sweet, looks great!
